### PR TITLE
Fix: Added online status to Doctor Connect

### LINKED ISSUE #7859: Online Status of the Doctors and Nurses are not shown in Doctor Connect
When the users(Doctor with Home Facility, Remote Doctor, Nurse - User types) are actively using CARE at instances the Online Status is not seen in the Doctor Connect. Instead the time after last Log-in is shown.

There's need to avail the ONLINE Status in the Doctor Connect so that it eases the user to Contact respective User without time leak.
Attaching the Supporting Image for work Ref.

image

Color Coding-Green showing- ONLINE/Green Bold Dot Should help this

### PR DESCRIPTION
**Pull request by Shelly011s**
## Proposed Changes

- Fixes #7859 
- Added a green dot to show the online status of staff nurses and doctor
Screenshot from 2024-08-06 09-05-00

@coronasafe/care-fe-code-reviewers @coronasafe/code-reviewers

## Merge Checklist

- [ ] Add specs that demonstrate bug / test a new feature.
- [ ] Update [product documentation](https://docs.coronasafe.network/coronasafe-care-documentation/architecture/architecture-and-layering-of-care).
- [ ] Ensure that UI text is kept in I18n files.
- [ ] Prep screenshot or demo video for changelog entry, and attach it to issue.
- [ ] Request for Peer Reviews
- [ ] Completion of QA

### CODE REVIEW

**Comment by samholics** [comment]
Tested Working 
@nihal467

File: src/Components/Facility/DoctorVideoSlideover.tsx [hunk 1]
```diff
@@ -243,7 +243,13 @@ function UserListItem({ user }: { user: UserAnnotatedWithGroup }) {
             // Show online icon based on last_login
             user.last_login &&
             Number(new Date()) - Number(new Date(user.last_login)) < 60000 ? (
```
Comment (rithviknishad) [inline src/Components/Facility/DoctorVideoSlideover.tsx]: @Shelly011s could you make use of the reusable utility fn. `isUserOnline` here instead, as that is what's being used elsewhere in the entire platform.

**Review by rithviknishad — CHANGES_REQUESTED** [review]

File: src/Components/Facility/DoctorVideoSlideover.tsx (hunk 1 above)
Comment (Shelly011s) [reply src/Components/Facility/DoctorVideoSlideover.tsx]: Updated

**Review by rithviknishad — APPROVED** [review]

**Comment by nihal467** [comment]
This will be working on the same logic as our user tab , LGTM

**Review by khavinshankar — APPROVED** [review]

**Comment by nihal467** [comment]
@Shelly011s nice work, feel free to join the community slack workspace : https://join.slack.com/t/rebuildearth/shared_invite/zt-2blycdt83-yvoJRsa10QDhE9WFiwoVkg

**Pull request merged**